### PR TITLE
Standardize configuration files across Fishjam JS repos

### PR DESCRIPTION
**Pull request by balins**
This PR is a part of JS repositories standardization process, which is to ensure all JS repositories have a similar set of scripts, same TS version, prettier config etc.

### CODE REVIEW

File: package-lock.json
Comment (mironiasty) [inline package-lock.json:1]: We are using `yarn` in other repose, so I think this file should be replaced with `yarn.lock`

File: .prettierrc
Comment (mironiasty) [inline .prettierrc:1]: As you are updating this file anyway, can it be same as here (+ plugin):
https://github.com/fishjam-dev/react-native-client-sdk/blob/main/.prettierrc
So we can have same config everywhere

File: tsconfig.json [hunk 1]
```diff
@@ -1,18 +1,24 @@
 {
+  "$schema": "https://json.schemastore.org/tsconfig",
+  "_version": "20.1.0",
```
Comment (mironiasty) [inline tsconfig.json:3]: Curious - is this required? I think VSCode (and other IDEs) recognise this file schema by default

File: examples/minimal/tsconfig.json [hunk 2]
```diff
@@ -1,19 +1,24 @@
 {
+  "$schema": "https://json.schemastore.org/tsconfig",
+  "_version": "20.1.0",
+
   "compilerOptions": {
```
Comment (mironiasty) [inline examples/minimal/tsconfig.json:5]: WDYT about extending main eslint/prettier/ts configs in example apps?

File: package-lock.json
Comment (balins) [reply package-lock.json:1]: We've been using `yarn` only in `react-native-client-sdk`, other use `npm`. But we can migrate all repos to use `yarn` if you wish.

File: .prettierrc
Comment (balins) [reply .prettierrc:1]: OK, but we need to update this with `printWidth` as well

File: tsconfig.json (hunk 1 above)
Comment (balins) [reply tsconfig.json:3]: You're right, it seems to not make any difference. I'll delete it.

File: examples/minimal/tsconfig.json (hunk 2 above)
Comment (balins) [reply examples/minimal/tsconfig.json:5]: Good idea!

File: package-lock.json
Comment (balins) [reply package-lock.json:1]: Nevermind, `yarn` would require a rewrite of the Release package CI workflow, so let's stick with `npm`.

**Review by mironiasty — APPROVED** [review]

**Pull request merged**